sycl/test-e2e/AddressSanitizer/common/debug.cpp

```diff
@@ -0,0 +1,17 @@
+// REQUIRES: linux, cpu
```

Comment:
rename to debug-message.cpp?

Reply:
rename to "kernel-debug", since it's used for debugging kernel only.